buildout.cfg

```diff
@@ -25,8 +26,16 @@ eggs =
     Pillow
     plone.app.robotframework [docs]
 
+[libdoc]
+recipe = zc.recipe.egg
+eggs = plone.app.robotframework [docs]
+entry-points = libdoc=robot.libdoc:libdoc_cli
+scripts = libdoc
+arguments = sys.argv[1:]
+
 [environment]
 zope_i18n_compile_mo_files = true
+ROBOT_SELENIUM2LIBRARY_RUN_ON_FAILURE = Capture page screenshot and log source
```

Comment:
@datakurre can you elaborate why this is not the default any longer? I first thought, plone.app.robotframework was broken and it took me a while to figure out why there are no screenshots on failures any longer.

Reply:
I recall Maurits removed it because of some CI issues. I’ll find the pull...